storage/fifocache/fifocacheSharded.go

```diff
@@ -140,6 +140,11 @@ func (c *FIFOShardedCache) MaxSize() int {
 	return c.maxsize
 }
 
+// Close does nothing for this cacher implementation
+func (c *FIFOShardedCache) Close() error {
+	return nil
```

Comment:
make clean the inner fields on `Close()` ?

Reply:
`Clear()` does this.